Assets/MRTK/SDK/Features/UX/Scripts/InteractiveElement/States/Definitions/InteractionType.cs

```diff
@@ -10,27 +10,27 @@ namespace Microsoft.MixedReality.Toolkit.UI.Interaction
     public enum InteractionType
     {
         /// <summary>
-        /// Does not support any form of input interaction.
+        /// Pointer independent interaction support.
         /// </summary>
-        None = 0,
+        Other = 0,
```

Comment:
Can you think of a case where you would still want "None"? I was kind of thinking you would keep none and add "Other" at the end.

Reply:
Here is a case: If a user creates a custom non input driven state (which is very likely), then interaction type could be None. 

I'll add None back and append Other.